Makefile

```diff
@@ -1,41 +1,139 @@
 VERSION=$(shell git describe --tags --abbrev=0)
 COMMIT=$(shell git rev-parse --short HEAD)
+DOCKER_COMPOSE_FILE=./.devcontainer/docker-compose.yaml
 
 ifeq ($(VERSION),)
 	VERSION="0.0.0"
 endif
 
+# Run gocyclo to check the cyclomatic complexities of the top 10 most complex functions
 gocyclo:
 	go run github.com/fzipp/gocyclo/cmd/gocyclo@latest -top 10 -ignore "_test|contract_" ./
 
+# Run gocognit to check cognitive complexities of the top 10 most complex functions
 gocognit:
 	go run github.com/uudashr/gocognit/cmd/gocognit@latest -top 10 -ignore "_test|contract_" ./
 
-# sort struct fields to reduce memory usage
+# Sort struct fields to reduce memory usage
 align:
 	go run github.com/dkorunic/betteralign/cmd/betteralign@latest -apply ./...
```

Comment:
And I propose adding a new `fmt` or `format` target to simultaneously execute `gocognit` and `align`, or integrate them into the existing `lint` target.

Reply:
I added a new target 'fmt' that execute both 2 commands to address this issue.